### DISCUSSION TITLE
Support plaseee!!!! 🥺🥺🥺🥺

### DISCUSSION THREAD

**Discussion by MRGalaxy507** [skyline-emu/skyline#1515] [Q&A] [answered]
there is a problem in gpu mali because sonic mania does not open you can only hear when it says sega and closes and chuphead only reaches the menu and exits every so often but celestial works normally could you help me or give an answer celestial runs at 60 fps I have a dimensity 800u and Mali-G57 MC3 gpu

**Comment by yhak22** [comment, marked as answer]
I hope they fix for mali ,
So u must wait anyway.

**Comment by dima-xd** [comment]
It will be fixed one day
